src/EnergyPlus/DXCoils.cc

```diff
@@ -9622,15 +9613,12 @@ Label50: ;
 					hTinwout = InletAirEnthalpy - ( 1.0 - SHR ) * hDelta;
 					LSOutletAirHumRat = PsyWFnTdbH( InletAirDryBulbTemp, hTinwout );
 					LSOutletAirDryBulbTemp = PsyTdbFnHW( LSOutletAirEnthalpy, LSOutletAirHumRat );
-					LSOutletAirRH = PsyRhFnTdbWPb( LSOutletAirDryBulbTemp, LSOutletAirHumRat, OutdoorPressure, RoutineNameLowSpeedOutlet );
```

Comment:
This line and next at 10772 are really the offenders since RH was calculated before the check for T < Tsat. Just delete them since they are redundant. The setting of OutletAirDryBulbTemp = OutletAirDryBulbTempSat isn't exactly correct since it changes the operating SHR slightly but it does conserve the deltaH calculated by the model. Since this warning shows up when inlet air conditions are near the saturation curve, it seems acceptable to allow the outlet air condition to be right next to the saturation curve.